composer.json

```diff
@@ -30,7 +30,7 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": ">=7.2",
```

Comment:
Would rather `^7.2.5 || ^8.0`.

Reply:
(just like on the main guzzle repo)

Reply:
I think the Guzzle repo is wrong too =)

Reply:
If I may give my 2c: `>=` is a pretty fragile constraint. It might cause a lot of hazzle for projects depending on this, if it turns out that e.g. PHP9 would not really work with this package as is (unepextected breaking). Yes, if it does work with PHP9, but no one is there to update the constraint, it makes it harder to install this package, but that's what `--ignore-platform-deps` is for.

Reply:
If this package is broken on PHP9, then we would get reports from the super cool bleeding edge users and we would be able to fix those bugs. 

Currently, everybody has to trust that the maintainers will use PHP9 and fix the bugs. I rather do open source as a community than people rely on my efforts.

Reply:
> I rather do open source as a community than people rely on my efforts

Agree. Guess we just have different PoV what that means/how that is achieved :) I rather have a stable working version that doesn't install on latest PHP but works guaranteed than leaving a surprising gambling open for the masses. If the bleeding edge users do try it out on latest platform, they can always create a PR that adds `|| ^9.0` and everyone wins ;)